archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ArchetypeEngineV2.java

```diff
@@ -99,11 +79,117 @@ public Path generate(InputResolver inputResolver,
         // resolve model  (full traversal)
         MergedModel model = MergedModel.resolveModel(script, context);
 
+        Map<String, String> userInputsMap = ContextSerializer.serialize(context);
+        String userInputs = userInputsMap.entrySet().stream()
+                                         .map(entry -> entry.getKey() + "=" + entry.getValue())
+                                         .collect(Collectors.joining(System.lineSeparator()));
+        model.node().add(new MergedModel.Value(model.node(), "user-inputs", 100, userInputs, null));
```

Comment:
Let's generate the file here directly instead of having to implement it in the archetype.

- Add a new option `--output-props-file` in the `init` command
- Add a new method in `ArchetypeInvoker.Builder` to pass the value from the `init` command
- Add a new method in `ArchetypeEngineV2.Builder` to pass the value from the invoker.
- Generate the props file inside `generate()`

Reply:
Can I use the method `store(...)` from `java.util.Properties` to write the user inputs to the file or it would be better to use `Mustache` for it?

Reply:
Using `java.util.Properties`.